packages/common/src/fs/findRootPath.test.js

```diff
@@ -0,0 +1,26 @@
+const path = require('path')
+const findRootPath = require('./findRootPath')
+const { REPO_ROOT_NAME } = require('../constants/repo')
+
+const getIsRootPath = rootPath => {
+  const rootPackagePath = path.join(rootPath, 'package.json')
+  /* eslint-disable import/no-dynamic-require, global-require */
+  const packageJson = require(rootPackagePath)
+  /* eslint-enable import/no-dynamic-require, global-require */
+  return REPO_ROOT_NAME === packageJson.name
+}
+
+describe('fs/findRootPath', () => {
+  it('Finds root path when no args provided', () => {
+    const rootPath = findRootPath()
+
+    expect(getIsRootPath(rootPath)).toBe(true)
+  })
+  it('Finds root path when path in node_modules provided', () => {
+    const rootPath = findRootPath({
+      startPath: path.join(__dirname, '../../node_modules/lodash'),
+    })
+
+    expect(getIsRootPath(rootPath)).toBe(true)
```

Comment:
Since we want to root of the repository we cant really know anything about the path. since a developer ex could name the root folder to something else than dina-collections. I have added some more tests in an upcoming pr.